vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-operators.xml

```diff
@@ -649,12 +649,18 @@
     
     <!-- opext:value($expr as xs:item(), $arg as xs:anyAtomicType?) as item() -->
     <operator name="opext:value">
-        <param name="expr" type="item()"/>
-        <param name="argExpr" type="xs:anyAtomicType"/>
+        <param name="expr" type="json-item()"/>
+        <param name="argExpr" type="item()"/>
         <return type="item()"/>
         <runtime type="scalar" class="org.apache.vxquery.runtime.functions.json.ValueScalarEvaluatorFactory"/>
     </operator>
 
+    <operator name="opext:keys">
```

Comment:
One is jdm:keys($o)[1] and the other is jn:keys($arg)[2]. I have change opext to jdm now.

[1] http://jsoniq.org/docs/JSONiqExtensionToXQuery/html-single/index.html#idm139680685942912
[2] http://jsoniq.org/docs/JSONiqExtensionToXQuery/html-single/index.html#idm139680622840960